server/vertx-server/src/main/scala/sttp/tapir/server/vertx/routing/PathMapping.scala

```diff
@@ -25,6 +26,14 @@ object PathMapping {
       case (None, path)         => router.route(path)
     }
 
+  private[vertx] def createOptionsRoute(router: Router, route: RouteDefinition): Option[Route] =
```

Comment:
I don't think I understand the purpose of this method. Is it "Options" as in the `OPTIONS` HTTP method? If so, why are we creating an options-route if the method is get/head/post/etc.?

Reply:
Based on [this](https://vertx.io/docs/vertx-web/java/#_route_match_failures) Vert.x will signal a 405 error if a route matches the path, but doesn’t match the HTTP Method. As I understand this is our case, so I've artificially added its counterpart `OPTIONS` route for the same path in order to get a chance to pass through. Do you see a better approach?